src/core/types/form-validator.ts

```diff
@@ -33,19 +33,23 @@ export type ValidationResult<Err> = Array<{
 export type ValidateIn<Err> = {
   fields: Array<FieldDescriptor<unknown, Err>>;
   trigger?: ValidationTrigger;
-  getValue: <P>(field: FieldDescriptor<P, Err>) => P;
+  getValue: GetValue;
   onFieldValidationStart?: (field: FieldDescriptor<unknown, Err>) => void;
   onFieldValidationEnd?: (field: FieldDescriptor<unknown, Err>) => void;
 };
 
+export type GetValue = {
+  <P>(field: FieldDescriptor<P, unknown>): P;
+  <P>(path: string): P;
+};
```

Comment:
wouldn't it be easier to always use the string version and remove FieldDescriptor overlaod?

Reply:
imho, this overload makes way more sense than path alternative
i like the fact that i don't have to always unpack opaque `FieldDescriptor` to get the value, and it covers most use cases
wdyt?